src/sections/TestProductSection/TestProductSection.tsx

```diff
@@ -5,6 +5,7 @@ import { Button } from '../../components/Button';
 import { Link } from '../../components/Link';
 import { SectionLayout } from '../../components/SectionLayout';
 import { TextAndMediaItem } from '../../components/TextAndMediaItem';
+import { List, ListItem } from '../../components/List';
```

Comment:
ESLint: `../../components/List` import should occur before import of `../../components/SectionLayout`

```suggestion
import { List, ListItem } from '../../components/List';
import { SectionLayout } from '../../components/SectionLayout';
import { TextAndMediaItem } from '../../components/TextAndMediaItem';
```